modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -3,76 +3,117 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes the processes for removing an Anypoint Platform PCE node from active service, and returning it to service when ready.
+== Before You Begin
 
-== Notes
+* Set up and test the backup procedure on all clusters after you complete installation.
+* Perform a backup before attempting node maintenance to preserve data in the event of a failure.
 
-[NOTE]
-To preserve your data in case of system failure, ensure that your system is backed up before attempting node maintenance.
-[NOTE]
-The steps below must be performed on a single Anypoint Platform PCE node at a time.  If maintenance is required on more than one node, the entire process must be completed on a single node before the next node is addressed.
+== Cluster and Application Updates
 
-== Drain the Node
+[IMPORTANT]
+The following steps must be performed on one Anypoint Platform Private Cloud Edition node at a time to minimize or eliminate component breakage during the update process. The entire process must be completed on each node before the next node is addressed. Verify the health status of all cluster components and applications after each node update before proceeding with the next node.
 
-Before any maintenance or restart of a node can be performed, it must be removed from the cluster and its workload reassigned to the other nodes in the cluster.
+=== Verify Cluster Status
 
-. Check if the firewalld service is running and enabled:
-+
-----
-service firewalld status
-----
-. If firewalld is running, stop, disable, and mask it:
+Verify that the cluster is healthy before starting the process. Otherwise, many of the assumptions made in this process are not valid.
+
+. Verify the output from the following commands:
 +
-----
-sudo systemctl stop firewalld
-sudo systemctl disable firewalld
-sudo systemctl mask firewalld
-----
-. Retrieve the *name* of target node:
+```
+gravity status
+etcdctl cluster-health
+```
+
+If the output from any of the two commands above mentions `degraded` state or `errors` please be sure to fix the mentioned issue, before proceeding.
+
+Please also verify that all Pods are running by running the commands below:
+```
+gravity get pod -o wide--all-namespaces | grep -vi Running
+```
+And if the command above lists any Pod, which will not be in "Running" state, then it means that it's probably having some issues and needs to be fixed.
+
+. Verify that replication works and is healthy in any replicated software (like Stolon and Cassandra).
+-- TODO link to Stolon and Cassandra troubleshooting here
```

Comment:
@eldios customers don't have access to the troubleshooting documentation for cassandra and stolon.
If we can't provide correct commands, we might as well leave this out.

Reply:
Removed. Debugging Stolon/Cassandra is a delicate matter and if they don't have access to the full KB then it's better not to deal with that.